security-perf.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Make sure to connect to the right cluster first.
+# Usage: source security-perf.sh 1.1.12 pre-release 1
+
+export RELEASE=$1
+export RELEASETYPE=$2
+export DNS_DOMAIN=perf.citadel.com
+export NAMESPACE="test-ns" 
+export NUM=10
+export CLUSTER=$(kubectl config current-context)
+TEST_TYPE=$3
+
+
```

Comment:
At the top of the script use the standard preamble, otherwise it has to be run from this directory.
Look at the top of https://github.com/istio/tools/blob/master/perf/istio-install/setup_istio.sh

for example